rcgen/tests/openssl.rs

```diff
@@ -540,3 +538,65 @@ fn test_openssl_pkcs1_and_sec1_keys() {
 	let pkcs8_ec_key_der = PrivateKeyDer::try_from(ec_key.private_key_to_pkcs8().unwrap()).unwrap();
 	KeyPair::try_from(&pkcs8_ec_key_der).unwrap();
 }
+
+#[test]
+#[cfg(feature = "x509-parser")]
+fn test_parse_certificate_with_multiple_domain_components() {
+	use rcgen::Ia5String;
+	use std::str::FromStr;
```

Comment:
Move these imports to the top of the file with the appropriate guard attributes, please.

Reply:
Done - I grouped them in a use block that is feature flagged. In case you prefer individual lines let me know.

Reply:
Done